pkg/cmd/pipeline/list_test.go

```diff
@@ -159,6 +159,129 @@ func TestPipelineList_only_pipelines_v1beta1(t *testing.T) {
 	golden.Assert(t, output, fmt.Sprintf("%s.golden", t.Name()))
 }
 
+func TestPipelineList_only_pipelines_no_headers_v1beta1(t *testing.T) {
+	pipelines := []pipelineDetails{
+		{"tomatoes", 1 * time.Minute, "namespace"},
+		{"mangoes", 20 * time.Second, "namespace"},
+		{"bananas", 512 * time.Hour, "namespace"}, // 3 weeks
+	}
+
+	nsList := []*corev1.Namespace{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "namespace",
+			},
+		},
+	}
+	version := "v1beta1"
+	clock := clockwork.NewFakeClock()
+	cs, pdata := seedPipelines(t, clock, pipelines, nsList)
+	cs.Pipeline.Resources = cb.APIResourceList(version, []string{"pipeline"})
+	tdc := testDynamic.Options{}
+	dc, err := tdc.Client(
+		cb.UnstructuredP(pdata[0], version),
+		cb.UnstructuredP(pdata[1], version),
+		cb.UnstructuredP(pdata[2], version),
+	)
+	if err != nil {
+		t.Errorf("unable to create dynamic clinet: %v", err)
+	}
+	p := &test.Params{Tekton: cs.Pipeline, Clock: clock, Kube: cs.Kube, Dynamic: dc}
+
+	pipeline := Command(p)
+	output, err := test.ExecuteCommand(pipeline, "list", "-n", "namespace", "--no-headers")
+	if err != nil {
+		t.Errorf("Unexpected error: %v", err)
+	}
+	golden.Assert(t, output, fmt.Sprintf("%s.golden", t.Name()))
+}
+
+func TestPipelineList_only_pipelines_all_namespaces_v1beta1(t *testing.T) {
+	pipelines := []pipelineDetails{
+		{"tomatoes", 1 * time.Minute, "namespace"},
+		{"mangoes", 20 * time.Second, "namespace"},
+		{"bananas", 512 * time.Hour, "namespace"}, // 3 weeks
+		{"tomates", 1 * time.Minute, "espace-de-nom"},
+		{"mangues", 20 * time.Second, "espace-de-nom"},
+		{"bananes", 512 * time.Hour, "espace-de-nom"}, // 3 weeks
+	}
+
+	ns := []*corev1.Namespace{
```

Comment:
You don't need this now that you are not checking for the namespace existing when `-A` is used.

Reply:
Updated